hooks/hookexecution/mocks_test.go

```diff
@@ -84,10 +85,14 @@ func (e mockRejectHook) HandleProcessedAuctionHook(_ context.Context, _ hookstag
 	return hookstage.HookResult[hookstage.ProcessedAuctionRequestPayload]{Reject: true}, nil
 }
 
+func (e mockRejectHook) HandleAllProcessedBidResponsesHook(_ context.Context, _ hookstage.ModuleInvocationContext, _ hookstage.AllProcessedBidResponsesPayload) (hookstage.HookResult[hookstage.AllProcessedBidResponsesPayload], error) {
+	return hookstage.HookResult[hookstage.AllProcessedBidResponsesPayload]{Reject: true}, nil
+}
+
 type mockTimeoutHook struct{}
 
 func (e mockTimeoutHook) HandleEntrypointHook(_ context.Context, _ hookstage.ModuleInvocationContext, _ hookstage.EntrypointPayload) (hookstage.HookResult[hookstage.EntrypointPayload], error) {
-	time.Sleep(2 * time.Millisecond)
+	time.Sleep(3 * time.Millisecond)
```

Comment:
Could you explain why this value was changed from 2 to 3?

Reply:
I guess, it was just a typo. Reverted to 2 milliseconds.